src/brogue/Combat.c

```diff
@@ -1600,7 +1600,8 @@ void killCreature(creature *decedent, boolean administrativeDeath) {
         }
     }
 
-    if (!administrativeDeath && (decedent->info.abilityFlags & MA_DF_ON_DEATH)) {
+    if (!administrativeDeath && (decedent->info.abilityFlags & MA_DF_ON_DEATH)
+        && ((rogue.patchVersion < 3) || !(decedent->bookkeepingFlags & MB_IS_FALLING))) {
```

Comment:
This means "don't spawn the DF if we die by falling" ? Does this mean the DF is not spawned at all, as opposed to being changed to spawn on the next floor?

Reply:
Yes, it doesn't spawn it at all.